packages/mgt-components/src/components/mgt-person-card/mgt-person-card.types.ts

```diff
@@ -81,7 +82,7 @@ export interface MgtPersonCardConfig {
   };
 }
 
-export type UserWithManager = User & { manager?: UserWithManager };
+export type UserWithManager = IUser & { manager?: UserWithManager };
```

Comment:
Do we ever use a Contact here? perhaps this should stay as the User type?

Reply:
Nope, will revert